### PR TITLE
Stats.js file has been created and all other modifications have done

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -106,7 +106,8 @@
                             <div class="col-6 col-lg-2 col-md-3 text-center color-white">
                                 <div class="coin-value-card" data-Speed="3000" data-aos="flip-right"
                                     data-aos-once="ture">
-                                    <h5 class="">0.00000000753379</h5>
+                                    <h5 class="" id="coin-price"></h5>
+                                    <!-- 0.00000000753379 -->
```
Comment (bobachef) [inline index.html:110]: please remove

File: js/stats.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+fetch('https://api.dex.guru/v1/tokens/0x3504de9e61fdff2fc70f5cc8a6d1ee493434c1aa-bsc')
```
Comment (bobachef) [inline js/stats.js:1]: can we actually wrap this in a function and use `async` `await` pattern?

**Review by bobachef — CHANGES_REQUESTED** [review]
some changes

File: index.html (hunk 1 above)
Comment (Shaharyar-saleem) [reply index.html]: removed

File: js/stats.js (hunk 2 above)
Comment (Shaharyar-saleem) [reply js/stats.js:1]: sure that is must to do, but let me complete the rest of 3 stats then will do

**Pull request closed without merging**